src/main/java/mate/jdbc/dao/impl/CarDaoImpl.java

```diff
@@ -0,0 +1,223 @@
+    }
+
+    @Override
+    public List<Car> getAllByDriver(Long driverId) {
+        String query = "SELECT id"
+                + " FROM cars c"
+                + " JOIN cars_drivers cd"
+                + " ON c.id = cd.driver_id"
+                + " WHERE cd.car_id = ? AND c.is_deleted = FALSE;";
+        List<Long> carIndexes = new ArrayList<>();
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement statement
+                         = connection.prepareStatement(query)) {
+            statement.setLong(1, driverId);
+            ResultSet resultSet = statement.executeQuery();
+            while (resultSet.next()) {
+                carIndexes.add(resultSet.getObject("id", Long.class));
+            }
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can't get cars from DB by driver id: "
+                    + driverId, e);
+        }
+        return carIndexes.stream()
+                .map(i -> get(i).orElseThrow(() ->
+                        new DataProcessingException("Can't get car from DB by id: " + i, null)))
+                .collect(Collectors.toList());
+    }
+
+    private Car parseCarWithManufacturer(ResultSet resultSet) throws SQLException {
+        Car car = new Car(0L, "");
+        car.setModel(resultSet.getString("model"));
+        Manufacturer manufacturer = new Manufacturer(
+                resultSet.getObject("manufacturer_id", Long.class),
+                resultSet.getString("name"),
+                resultSet.getString("country")
+        );
+        car.setManufacturer(manufacturer);
+        car.setId(resultSet.getObject("car_id", Long.class));
+        return car;
+    }
+
+    private List<Driver> getDriversForCar(Long carId) {
+        String driversQuery = "SELECT id, name, license_number"
+                + " FROM drivers d"
+                + " JOIN cars_drivers cd"
+                + " ON d.id = cd.driver_id"
+                + " WHERE cd.car_id = ? AND d.is_deleted = FALSE;";
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement statement = connection.prepareStatement(driversQuery)) {
+            statement.setLong(1, carId);
+            ResultSet resultSet = statement.executeQuery();
+            List<Driver> drivers = new ArrayList<>();
+            while (resultSet.next()) {
+                drivers.add(parseDriverFromResultSet(resultSet));
+            }
+            return drivers;
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can't find drivers in DB by car id: " + carId, e);
+        }
+    }
+
+    private Driver parseDriverFromResultSet(ResultSet resultSet) throws SQLException {
+        return new Driver(
+                resultSet.getObject("id", Long.class),
+                resultSet.getString("name"),
+                resultSet.getString("license_number"));
+    }
+
+    private void refreshDrivers(Car car) {
```

Comment:
do not forget to rename it after the fix of the comment above

Reply:
ok